src/_data/sites/ja.yml

```diff
@@ -0,0 +1,548 @@
+---
+#------------------------------------------------------------------------------
+# English Site Details
+# The primary ESLint site that is hosted at eslint.org
+# Author: Nicholas C. Zakas
+#------------------------------------------------------------------------------
+
+#------------------------------------------------------------------------------
+# Global Settings
+#------------------------------------------------------------------------------
+
+language:
+  code: ja
+  flag: 🇯🇵
+  name: Japanese
+locale: ja-JP
+hostname: ja.eslint.org
```

Comment:
Would this make sense instead? Unsure what is common for Japanese language sites. 

```suggestion
hostname: jp.eslint.org
```

Reply:
we use https://ja.reactjs.org/ in reactjs.org docs

Reply:
Okay, ja.eslint.org it is. 👍